rust/src/sip/sip.rs

```diff
@@ -298,6 +392,31 @@ pub unsafe extern "C" fn rs_sip_probing_parser_ts(
     return ALPROTO_UNKNOWN;
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn rs_sip_probing_parser_tcp_ts(
```

Comment:
Could we rather have pattern matching instead of probing parser for SIP ?

Reply:
I think there is already a ticket for that: https://redmine.openinfosecfoundation.org/issues/5047

May it be the next step after that this PR will be merged?